javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,33 @@
+//Ez Namey (Startup name generator) Optional
+
+let firstWords = [
+  "Easy",
+  "Awesome",
+  "Corporate",
+  "Creative",
+  "Let",
+  "Make",
+  "Space",
+  "Fox",
+  "Hi",
+  "Great",
+];
+let secondWords = [
+  "Peace",
+  "Deal",
+  "partner",
+  "ideas",
+  "dream",
+  "possible",
+  "around",
+  "lox",
+  "there",
+  "team",
+];
+
+let startupName = () =>
+  `${firstWords[Math.floor(Math.random() * firstWords.length)]} ${
+    secondWords[Math.floor(Math.random() * secondWords.length)]
+  }`;
+
+console.log(startupName());
```

Comment:
You don't need to use arrow function here. you pick one random string from first array and repeat the same for the second one. You should also print out the startup name length, that is part of the question. And you could use `const` to declare all the variables here.